mandatory/3-playing-computer.js

```diff
@@ -28,13 +37,13 @@ const f2 = function (a, b) {
 
 console.log(x);
 console.log(a);
-console.log(b);
+// console.log(b);
```

Comment:
Maybe you can delete this line instead of commenting it out. They say to add commented lines if you're leaving notes to understand the code better, otherwise you can just delete it.